### ISSUE TITLE
The browser & dev-server builders are not calling modifyCopyAssets

### ISSUE THREAD

**Issue by DavainWhite** [ZachJW34/nx-plus#84]
Just spent an hour or 6 looking trough the plugin and found that the `modifyCopyAssets` function was not being called in the webpack for the browser & dev-server builders causing the assets in the public directory to not be added to the dist folder on serve / build.

you ca see in these images that the favicon.ico &  the site manifest were copped over on build (because I added the custom `modifyCopyAssets2` function to the `inlineOptions` array; those files are omitted if I comment them out)
![image](https://user-images.githubusercontent.com/40371122/89963988-1a310480-dc17-11ea-879b-506026d8f95f.png)
![image](https://user-images.githubusercontent.com/40371122/89964074-4f3d5700-dc17-11ea-9aff-a203f8737098.png)

I'll attached the mangled download of the nx-plus repo
the only files I've chnaged to get this to work for me we the 
**libs\vue\src\builders\browser\builder.ts** ( added `modifyCopyAssets2` function )
**libs\vue\src\webpack.ts** ( created `modifyCopyAssets2` function )
**apps\vue-e2e\tests\vue.test.ts** ( removed all unnecessary tests )
**libs\vue\src\schematics\application\files\public\site.webmanifest.template** (added a non standard template file to see if the copy-webpack-plugin would have any issues)

To reproduce this patch / fix / error.
1). run nx e2e vue-e2e to generate the tmp dist for the app
2). uncomment out the `modifyCopyAssets2 (config, options, context, projectRoot);` line in the **libs\vue\src\builders\browser\builder.ts** and run nx e2e vue-e2e again to see if all the same files are present in the dist folder.

TL;DR add `modifyCopyAssets` to all the builder.ts files because the dev-server * browser builders are not adding static public files to the dist folder.
![image](https://user-images.githubusercontent.com/40371122/89964677-d0491e00-dc18-11ea-9482-f0ef9d7b8ff5.png)

[nx-plus.zip](https://github.com/ZachJW34/nx-plus/files/5060148/nx-plus.zip)

**Comment by DavainWhite** [comment]
@BuckyMaler were there any errors in what I found in the post above ?

**Comment by ZachJW34** [comment]
@DavainWhite Do you develop on a Windows machine? I just submitted a PR that fixes assets not being copied on Windows machines. #88

**Comment by DavainWhite** [comment]
@ZachJW34 yes, windows 10

**Comment by ZachJW34** [comment]
@DavainWhite I believe this should fix your issue then. Try checking out the branch and adding some assets. If you cloned the repo, you can check out the branch by running `git fetch origin pull/88/head:fix-vue-windows-assets && git checkout fix-vue-windows-assets`

**Comment by DavainWhite** [comment]
@ZachJW34 removed all tests but the build and build subdir for time's sake 
![image](https://user-images.githubusercontent.com/40371122/90321783-e574c400-df1a-11ea-8e0b-17f9f8140584.png)
[vue.test.zip](https://github.com/ZachJW34/nx-plus/files/5079509/vue.test.zip)

Added custom template files site.manifest and it worked
![image](https://user-images.githubusercontent.com/40371122/90321846-a6933e00-df1b-11ea-8c6a-9bb8a339cc62.png)
[vue.test.zip](https://github.com/ZachJW34/nx-plus/files/5079514/vue.test.zip)

**Comment by DavainWhite** [comment]
@ZachJW34 looks good on windows, is there a scheduled date for the next npm release ?
